src/hermes/commands/init/base.py

```diff
@@ -173,8 +181,21 @@ def __init__(self, parser: argparse.ArgumentParser):
             "deposit_extra_files": "",
             "push_branch": "main"
         }
+        self.hermes_toml_data = {
+            "harvest": {
+                "sources": ["cff"]
+            },
+            "deposit": {
+                "target": "invenio_rdm",
+                "invenio_rdm": {
+                    "site_url": "",
+                    "access_right": "open"
+                 }
+            }
+        }
```

Comment:
This feels like we should use `hermes.commands.base._HermesSettings` instead. But I don't see how 😅